main.py

```diff
@@ -154,22 +177,20 @@ def count_beer(travel_df):
     Count how many types of beer each brewery has
     """
     number_of_beers = 0
-    for row in travel_df.index:
-        br_id = travel_df.loc[row]["brewery_id"]
-        beers = beer_count_df.loc[br_id].values[0]
+    for row in travel_df.itertuples():
+        beers = beer_count_df.loc[row.brewery_id].values[0]
```

Comment:
Can you have beer count value in travel_df in advance?
Same as comments above

Reply:
Fixed